website/docs/d/bfd_multihop_interface_policy.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_bfd_multihop_interface_policy"
+sidebar_current: "docs-aci-data-source-bfd_multihop_interface_policy"
+description: |-
+  Data source for ACI BFD Multihop Interface Policy
+---
+
+# aci_bfd_multihop_interface_policy #
+
+Data source for ACI BFD Multihop Interface Policy
+
+
+## API Information ##
+
+* `Class` - bfdMhIfPol
+* `Distinguished Name` - uni/tn-{tn_name}/bfdMhIfPol-{policy_name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Policies -> Protocol -> BFD Multihop -> Interface Policies
+
+
+
+## Example Usage ##
+
+```hcl
+data "aci_bfd_multihop_interface_policy" "example" {
+  tenant_dn = aci_tenant.example.id
+  name      = "example"
+}
+```
+
+## Argument Reference ##
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object. Type: String.
+* `name` - (Required) Name of the  BFD Multihop Interface Policy object. Type: String.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the BFD Multihop Interface Policy object. Type: String.
+* `annotation` - (Read-Only) Annotation of the BFD Multihop Interface Policy object. Type: String.
+* `name_alias` - (Read-Only) Name Alias of the BFD Multihop Interface Policy object. Type: String.
+* `admin_state` - (Read-Only) Enable Disable sessions. The administrative state of the object or policy. Type: String.
```

Comment:
"Enable Disable sessions." can be removed to be consistent with the other description in the resource doc.

Reply:
resolved.